andela/tesla-ah
# #166978211 mockup implementation

### what does this PR do?
- Enable users to see a presentable user interface that suits them.
### Description of the task to be completed
The user should able to interact with the app through an interface.
### How should it manually tested
- Clone this branch, cd into it
- Install dependencies by running `yarn `
- Run scss: `yarn watch-sass`

- Then put the localhost in the browser to see what is happening: 
**for login**: `http://127.0.0.1:5500/UI/html/login.html `
**for signup**:  `http://127.0.0.1:5500/UI/html/signup.html `


### Pivotal Tracker Story
166978211
